addon/components/rdfa/editor-toolbar.hbs

```diff
@@ -64,7 +64,7 @@
       </button>
       {{/if}}
       {{#if @showIndentButtons}}
-      <button type="button" class="say-toolbar__button" {{on "click" this.insertUnindent}} data-test-button-id="insert-unindent">
+      <button type="button" class="say-toolbar__button {{if this.isInList "is-active"}}" {{on "click" this.insertUnindent}} data-test-button-id="insert-unindent">
```

Comment:
This seems counter intuitive, the button is not active when in a list. I think you want `class={{unless this.isInList "disabled"}}`, as well as a disabled state on the button itself

Reply:
I think this is changed in the indent/unindent prs, so I'd leave it as is and see how it behaves after all merges